plat/mediatek/common/mtk_sip_svc.h

```diff
@@ -42,19 +42,30 @@
 #define MTK_SIP_SVC_VERSION_MAJOR	0x0
 #define MTK_SIP_SVC_VERSION_MINOR	0x1
 
+#define SMC_AARCH64_BIT		0x40000000
+
 /* Number of Mediatek SiP Calls implemented */
-#define MTK_SIP_NUM_CALLS		4
+#define MTK_SIP_NUM_CALLS		5
```

Comment:
I do not count it right in compile time. 
So far, the upstream SMC call number should be 4(since mt8173 use MTK_SIP_SET_AUTHORIZED_SECURE_REG_ENABLE and mt6795 use MTK_SIP_KERNEL_BOOT_ENABLE). So far there is no SMC implementation in mediatek_plat_sip_handler.